app/models/articles/feeds.rb

```diff
@@ -78,6 +79,10 @@ def self.lever_catalog
       LEVER_CATALOG
     end
 
+    def self.number_of_hours_to_offset_users_latest_article_views
+      (ApplicationConfig["NUMBER_OF_HOURS_TO_OFFSET_USERS_LATEST_ARTICLE_VIEWS"] || 18).to_i
+    end
```

Comment:
Isn't this a bit redundant? I believe the class var is enough in this case

Reply:
@maestromac the reason for this variable is how the class var is loaded relative to how and when the code is evaluated. Class var is or can be first loaded before the ENV var (and the tests play this out).

We have this same issue in other places in the same class, i.e.

```ruby
    def self.lever_catalog
      LEVER_CATALOG
    end
```

But you are right in that the way it was written was overly redundant and I have made a change to clean that up.